src/components/shared/Caret.tsx

```diff
@@ -1,21 +1,21 @@
-import { Icon, IIconProps } from '@stoplight/ui-kit';
+import { Icon, IIconProps } from '@stoplight/mosaic';
 import * as React from 'react';
 
 export interface ICaret {
   isExpanded: boolean;
   style?: React.CSSProperties;
-  size?: IIconProps['iconSize'];
+  size?: IIconProps['size'];
 }
 
 export const Caret: React.FunctionComponent<ICaret> = ({ style, size, isExpanded }) => (
   <span
-    className="absolute flex justify-center cursor-pointer p-1 rounded hover:bg-darken-3"
+    className="absolute flex justify-center cursor-pointer p-1"
     role="button"
     style={style}
   >
     <Icon
       iconSize={size}
-      icon={isExpanded ? 'caret-down' : 'caret-right'}
+      icon={['fas', isExpanded ? 'chevron-down' : 'chevron-right']}
```

Comment:
`fas` is the default - I suggest omitting it and just passing props similar to before (but with chevron) e.g. `icon={isExpanded ? 'chevron-down' : 'chevron-right'}`